web/client/utils/__tests__/FilterUtils-test.js

```diff
@@ -484,7 +484,7 @@ describe('FilterUtils', () => {
         // this is a workarround for this issue : https://github.com/geosolutions-it/MapStore2/issues/1263
         // please remove when fixed
         FilterUtils.nsplaceholder = "ogc";
-        FilterUtils.setOperatorsPlaceholders("{namespace}", "ogc");
+        FilterUtils.setOperatorsPlaceholders("{namespace}", FilterUtils.nsplaceholder);
```

Comment:
Not this way: the FilterUtils.setOperatorsPlaceholders function should have a default for the second argument:

setOperatorsPlaceholders: function(placeholder, replacement = "ogc") {
   ...
}